e2e/extensions.e2e.spec.ts

```diff
@@ -283,7 +283,7 @@ test.describe.serial('Extensions', () => {
       test('ddClient.docker.listImages', async() => {
         const options = {
           digests:   true,
-          namesapce: isContainerd ? NAMESPACE : undefined,
+          namespace: isContainerd ? NAMESPACE : undefined,
```

Comment:
??

Reply:
Suggests to me that there's an invalid test lurking here.

Reply:
This is staying -- see issue #5031